### PR TITLE
Update rockspec and README, remove Makefile

### PR DESCRIPTION
**Pull request by rolandlo**
Updated the rockspec (rockspec-format, labels, dependencies, test, copy-directories).

`luaffi-tkl` is now a dependency of `lua-vips`. LuaJIT users will have to exhibit this module as provided by the VM by editing the `config-5.1.lua` or by running `luarocks config --lua-version=5.1 rocks_provided.luaffi-tkl 2.1-1`. Ideally this would be listed in https://github.com/luarocks/luarocks/blob/c16bdbc4cbb98c560fb7e8d4895d4d84afca2eb9/src/luarocks/util.lua#L599-L604, so LuaJIT user's wouldn't have to do it manually, but that would only work for those with an up to date luarocks then.

When running `luarocks test` in the `lua-vips` directory the test-dependency `busted` is automatically installed, if it is not found.

Updated the README (where `lua-vips` works and where it is tested, how to install on Windows MinGW-w64).

Removed the Makefile, as it is not needed any longer. @RiskoZoSlovenska What do you think about it?

### CODE REVIEW

File: lua-vips-1.1-10.rockspec [hunk 1]
```diff
@@ -34,5 +47,6 @@ build = {
        ["vips.Image"] = "src/vips/Image.lua",
        ["vips.Image_methods"] = "src/vips/Image_methods.lua",
        ["vips.Interpolate"] = "src/vips/Interpolate.lua"
-   }
+   },
+   copy_directories = { "example" }
```
Comment (RiskoZS) [inline lua-vips-1.1-10.rockspec]: Is there a point in copying the `example` directory? As in, do you expect users to navigate into their Luarocks install directory to run the example code?

Comment (rolandlo) [reply lua-vips-1.1-10.rockspec]: Yeah, not sure about it. I thought it was good practice to copy directories like "docs" or "examples". Looking at some widely used modules it doesn't look so. I'll remove it then.

**Comment by RiskoZS** [comment]
Looks good to me; nice work!

**Comment by rolandlo** [comment]
Thanks for the review @RiskoZoSlovenska

**Pull request merged**